controller/characterController.go

```diff
@@ -0,0 +1,102 @@
+package controller
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/carRub/academy-go-q12021/model"
+	"github.com/gorilla/mux"
+	"github.com/unrolled/render"
+)
```

Comment:
```suggestion
import (
	"fmt"
	"net/http"
	"strconv"

	"github.com/carRub/academy-go-q12021/model"
	
	"github.com/gorilla/mux"
	"github.com/unrolled/render"
)
```
Dependencies must be separated by Golang's own libraries, then your own project dependencies, and lastly your 3rd party dependencies.